{
  "name": "catch-finally-decorator",
  "version": "1.0.4",
  "description": "Let you handle exceptions with just one annotation, including finally blocks",
  "main": "dist/index.js",
  "typings": "dist/index.d.ts",
  "scripts": {
    "build": "rollup --config",
    "test": "npm run build && jest"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/codewithbanji/catch-decorator"
  },
  "keywords": [
    "catch",
    "decorator",
    "javascript",
    "errors",
    "handling"
  ],
  "author": "Enkot",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/codewithbanji/catch-decorator/issues"
  },
  "homepage": "https://github.com/codewithbanji/catch-decorator#readme",
  "devDependencies": {
    "@types/jest": "^22.2.3",
    "jest": "^23.6.0",
    "rollup": "^0.59.3",
    "rollup-plugin-license": "^0.7.0",
    "rollup-plugin-typescript2": "^0.14.0",
    "ts-jest": "^22.4.6",
    "ts-loader": "^2.0.3",
    "typescript": "^2.7.2"
  }
}
